### ISSUE TITLE
File not found for module

### ISSUE THREAD

**Issue by David-OConnor** [intellij-rust/intellij-rust#5742]
## Environment

* **IntelliJ Rust plugin version:** 0.3.126.3220-201
* **Rust toolchain version:** rustc 1.46.0-nightly (f781babf8 2020-07-01)
* **IDE name and version:** PyCharm 2020.1.2
* **Operating system:** Windows 10

## Problem description
Not recognizing modules: `File not found for module 'mymod' [E0583]`

## Steps to reproduce
Create `main.rs`
Create `mymod.rs` in the same folder.

In `main.rs`:
```rust
mod `mymod`;
```
The module text in that line will be red, and the editor won't recognize fns, structs etc imported from that module.

**Comment by Undin** [comment]
I can't reproduce it.
Could you provide an exact code example where the plugin can't find module?

**Comment by NTmatter** [comment]
* **IntelliJ Rust plugin version**: 0.3.127.3249-201
* **Rust toolchain version**: rustc 1.45.0 (5c1f21c3b 2020-07-13)
* **IDE name and version**: CLion 2020.1.3
* **Operating System**: Fedora 32

Cargo.toml:
```toml
[package]
name = "intellij_rust_5742"
version = "0.1.0"
authors = ["Johnathan Doe <jdoe@example.com>"]
edition = "2018"

[dependencies]
```

src/main.rs:
```rust
mod example;
use example::foo;

fn main() {
    println!("Hello, world: {}", foo());
}
```

src/example/mod.rs:
```rust
pub fn foo() -> u8 {
    1 + 1
}
```

If I open mod.rs, I get a warning stating `File is not included in module tree, analysis is not available`

**Comment by NTmatter** [comment]
Hang on, I just reopened the project with a different path and there are no issues.

My home directory has a symlink to a separate volume: `/home/jdoe/2020 -> /DATA/archives/2020/`

The project's apparent path is `/home/jdoe/2020/ij_rust_5472` which maps to `/DATA/archives/2020/ij_rust_5472`

If I open the symlinked version in my homedir, I start seeing errors. If I open the physical path, no problems are reported.

**Comment by vlad20012** [comment]
Duplicate #5861, fixed in #6075
